modules/orchestrator/proc-install-rhdh-orchestrator-airgapped-env-using-helm-chart.adoc

```diff
@@ -0,0 +1,116 @@
+
+. Mirror the `redhat-developer-hub-orchestrator-infra` chart images to the local registry.
+
+. Create an `ImageSetConfiguration` file for `oc-mirror`. Because `oc-mirror` does not automatically mirror all images required by the Serverless Logic Operator, you must use an `ImageSetConfiguration` file to include them as shown in the following example:
++
+[source,subs="+attributes,+quotes"]
+----
+apiVersion: mirror.openshift.io/v2alpha1
+kind: ImageSetConfiguration
+mirror:
+  additionalimages:
+  - name: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-db-migrator-tool-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:{rhoserverless-version}.0
+
+  helm:
+    repositories:
+      - name: openshift-charts
+        url: https://charts.openshift.io
+        charts:
+          - name: redhat-developer-hub
+            version: "{product-bundle-version}"
+          - name: redhat-developer-hub-orchestrator-infra
+            version: "{product-bundle-version}"
+----
++
+Alternatively, you can use the podman commands to find the missing images and add them to the `additionalimages` list if the versions change:
++
+[source,subs="+attributes,+quotes"]
+----
+IMG=registry.redhat.io/openshift-serverless-1/logic-operator-bundle:{rhoserverless-version}.0-8
+mkdir local-manifests-osl
+podman create --name temp-container "$IMG" -c "cat /manifests/logic-operator-rhel8.clusterserviceversion.yaml"
+podman cp temp-container:/manifests ./local-manifests-osl
+podman rm temp-container
+yq -r '.data."controllers_cfg.yaml" | from_yaml | .. | select(tag == "!!str") | select(test("^.*\\/.*:.*$"))' ./local-manifests-osl/manifests/logic-operator-rhel8-controllers-config_v1_configmap.yaml
+----
+
+. Mirror the images in the `ImageSetConfiguration.yaml` file by running the `oc-mirror` command. For example:
++
+[source,subs="+attributes,+quotes"]
+----
+oc-mirror --config=imagesetconfiguration.yaml docker://<registry URL:port> --workspace file://<workspace folder> --authfile /path/to/authfile  --v2
+cd <workspace folder>/working-dir/cluster-resources/
+oc apply -f .
+----
```

Comment:
Same suggestion as for the Operator: we should ideally split this into 2 cases: partially vs fully disconnected